pkg/gofr/datasource/pubsub/mqtt/mqtt.go

```diff
@@ -338,19 +341,25 @@ func (m *MQTT) Unsubscribe(topic string) error {
 		close(sub.msgs)
 		delete(m.subscriptions, topic)
 	}
-	m.mu.RUnlock()
+
+	return nil
+}
+
+func (m *MQTT) Close(_ context.Context) error {
+	m.Client.Disconnect(closeTimeout)
```

Comment:
Sure I have included it in MQTT Config, so the user can define it while passing configs to New method of mqtt package, but when initializing it using GoFr env variables (using container) we have set the default value to 250 milliseconds so that user don't have to worry when using default pub-sub configurations.

Reply:
We don't want to create a env variable for closeTimeout, as firstly we only can't create it for MQTT we shall also then provide support of this env for other pubsub brokers or maybe other DB's wherever applicable.

Currently when creating a **new mqtt client through container** we are using default close timeout value else what is the value set when using `New` to create a mqtt client, we are using that.